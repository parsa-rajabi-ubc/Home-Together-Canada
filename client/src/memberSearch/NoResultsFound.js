/**
 * @Author:     Jeff Hatton
 * @Created:    2021.1.23
 *
 * @Description: returns stateless functional component to display a message
 *
 */

import React from 'react';

function NoResultsFound(){
    return(
        <div>
            No Results found
        </div>
    )
}

export default NoResultsFound;