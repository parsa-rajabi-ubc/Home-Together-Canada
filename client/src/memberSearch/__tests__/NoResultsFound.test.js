/**
 * @Author:     Jeff Hatton
 * @Created:    2021.1.23
 *
 * @Description: Snapshot test for member search results should render properly;
 *
 */

import React from 'react';
import renderer from  'react-test-renderer'
import NoResultsFound from "../NoResultsFound";

describe('MemberSearchResults', () => {
    describe('Container test', () => {
        it('should match snapshot test with an empty array', () => {
            //when
            const component = renderer.create(<NoResultsFound/>);
            const tree = component.toJSON();
            //then
            expect(tree).toMatchSnapshot();
        });
    });
});