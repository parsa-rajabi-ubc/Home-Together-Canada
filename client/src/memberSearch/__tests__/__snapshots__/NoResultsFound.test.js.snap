// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MemberSearchResults Container test should match snapshot test with an empty array 1`] = `
<div>
  No Results found
</div>
`;
